eventhubs/client/azure-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubProducer.java

```diff
@@ -52,6 +52,28 @@
  * </ol>
  * </p>
  *
+ * <p><strong>Create a producer that routes events to any partition</strong></p>
+ * To allow automatic routing of messages to available partition, do not specify the {@link
+ * EventHubProducerOptions#partitionId() partitionId} when creating the {@link EventHubProducer}.
+ *
+ * {@codesnippet com.azure.messaging.eventhubs.eventhubproducer.instantiate}
+ *
+ * <p><strong>Create a producer that pushes events to partition "foo" with a timeout of 45 seconds.</strong></p>
```

Comment:
nit:  "pushes" seems a bit unusual.   Maybe considers "publishes" or "sends" or "delivers"....

Reply:
I'll stick with the publishes/producers theme. My producer publishes movies.